thunder/executors/transformer_engineex.py

```diff
@@ -290,7 +290,7 @@ def _get_quantizers(self, is_grad_enabled):
         grad_output_quantizer = None
         output_quantizer = None
         input_quantizer = self.quantizers["scaling_fwd"][tex.FP8FwdTensors.GEMM1_INPUT]
-        input_quantizer.internal = False
+        input_quantizer.internal = True
```

Comment:
Leaving an unresolved comment so that this isn't merged before I check this on the various devices.